.github/workflows/specs.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: [ jruby, ruby ]
+        ruby-version: [ jruby-9.3.8.0, ruby ]
```

Comment:
I have freezed jruby version to `jruby-9.3.9.0` in master branch so you can remove this change and this branch will still pass the build.